Automattic/wp-calypso
# Offer Reset: add the Pricing page footer

#### Changes proposed in this Pull Request

* Copy jetpack.com footer to cloud.jetpack.com/pricing.

#### Things to keep in mind

* The background color is not white as it should. This is being added in https://github.com/Automattic/wp-calypso/pull/45369.
* The color of the links is not the right one. I expect this to be fixed along with fonts in another PR.

#### Testing instructions

* Run this PR with `yarn start-jetpack-cloud`.
* Visit `http://jetpack.cloud.localhost:3000/pricing`.
* Go to the bottom of the page and check the following (always comparing against the footer in `https://jetpack.com/pricing/`:
* Links are valid and point to the right place (sorry for making you open 1000 tabs).
* Links are grouped in the same way.
* The UI responds similarly as it does in jetpack.com when you change the screen width. It is expected that some differences may appear since I made the link groups match the current plans grid and that changed things a bit.
* Links columns go from 4, to 2, to 1, if you decrease the screen size. 
* Don't forget to test the behavior of the Jetpack logo under different viewport sizes.

Fixes 1169247016322522-as-1192011804172289

#### Demo

![image](https://user-images.githubusercontent.com/3418513/92138767-3292da00-ede5-11ea-892b-35c421a5af7b.png)

![image](https://user-images.githubusercontent.com/3418513/92138829-49393100-ede5-11ea-9358-44530a1df03e.png)

![image](https://user-images.githubusercontent.com/3418513/92138873-59511080-ede5-11ea-9759-4f74169cfbbc.png)
